includes/fbs.availability.inc

```diff
@@ -12,29 +11,21 @@
  *   Array of ting object ids (faust).
  *
  * @return array
- *   Availability info.
+ *   Avaliability info.
  */
 function fbs_availability_items($provider_ids) {
   $avails = array();
-  $service = fbs_service();
-  if ($service) {
-    try {
-      $avails = $service->Catalog->getAvailability(
-        $service->agencyId,
-        $provider_ids
-      );
-    }
-    catch (Exception $e) {
-      watchdog_exception('fbs', $e);
-    }
+
+  try {
+    $avails = fbs_service()->Catalog->getAvailability(fbs_service()->agencyId, $provider_ids, variable_get('fbs_holdings_branches_blacklist', array()));
+  }
+  catch (Exception $e) {
+    watchdog_exception('fbs', $e);
   }
 
   $result = array();
   foreach ($avails as $item) {
-    $result[$item->recordId] = array(
-      'available' => $item->available,
-      'reservable' => $item->reservable,
-    );
+    $result[$item->recordId] = array('available' => $item->available, 'reservable' => $item->reservable);
```

Comment:
Certain places have each array item on it's own line, others not.
Having multiple array values on each line helps readability.